### DISCUSSION TITLE
Template with esbuild, pnpm, typescript and prettier

### DISCUSSION THREAD

**Discussion by lampewebdev** [excaliburjs/Excalibur#2151] [Show and tell]
I have created a template from my own setup with
- esbuild (build tool)
- pnpm (package manager)
- Typescript (language)
- prettier (code formatter)

You can find that template here:
https://github.com/lampewebdev/excalibur-pnpm-esbuild-typescript-prettier/tree/main/src

**Comment by eonarheim** [comment]
@lampewebdev Awesome! I'll be checking this out later!
